English translation for 2022.02 release

sungchang79 opened the pull request:
- Translate new content
- Modify unnatural translation

sungchang79 committed:
English translation for 2022.02 release
- Translate new content
- Modify unnatural translation

euijoong-nhntoast commented:
@sungchang79 
우선, 번역 감사드립니다.

release_notes.md의 16라인, 수정된 영문을 해석하면, '입력 이미지 형식이 다양하다' 인 것 같은데요...
입력 가능한 이미지 형식이 몇가지 있었고, 여기에 몇가지 형식이 추가된 거라서 수정해 주신 영문이 맞는건가? 싶은 의문이 살짝 듭니다.
'입력 가능한 이미지 형식이 확대 되었다'는 것을 영어로는 수정해 주신 것 처럼 표현하는게 더 의미가 맞는 걸까요? ^^;
궁금해서 문의 드립니다. 확인 부탁드립니다. ^^

sungchang79 commented:
@euijoong-nhntoast 팀장님,

명사구 형태('diversification of')를 동사구 형태(diversified ~)로 바꾸고, 한글 원문에 가깝게 '업로드 가능한 이미지 포맷(uploadable image formats)를 '입력 (가능한) 이미지 포맷(input image formats)'으로 바꾼 것입니다.

우선 Release notes는 변경 사항에 해당하는 동사구로 표현하는 것이 일반적이라서(예: Added a feature to ~, Fixed an issued where ~) 스타일 일관성을 위해 수정하였습니다. '입력 이미지 형식이 다양하다'의 의미는 아니며, '입력 이미지 형식을 다양화했다'로 보는 편이 타당하겠습니다.

```
(기존) * [API] Diversification of uploadable image formats
(변경) * [API] Diversified input image formats
```

더 개선하려면 한글 원문을 '입력 이미지 형식으로 ~를 추가함'과 같이 구체적으로 표현하는 편이 모든 언어에 걸쳐 더 나은 결과로 이어질 수 있겠습니다.

euijoong-nhntoast commented:
@sungchang79 수석님, 자세한 설명 감사드립니다.
역시 번역은 더 어렵네요. ^^
merge 하겠습니다.
